randovania/games/dread/logic_database/Ghavoran.txt

```diff
@@ -1726,10 +1724,8 @@ Extra - asset_id: collision_camera_024
           Morph Ball and Knowledge (Intermediate) and Disabled Door Lock Randomizer
           Any of the following:
               Disabled Power Bomb Limitations and Lay Power Bomb
-              All of the following:
-                  Shoot Charge Beam
-                  # The pseudo for this is a melee pseudo next to the tunnel
-                  Wave Beam or Pseudo-Wave Beam (Beginner)
+              # The pseudo for this is a melee pseudo next to the tunnel
+              Shoot Diffusion Beam
```

Comment:
I would probably add back the pseudo for this.

Reply:
Done